web/client/components/data/query/GeometryDetails.jsx

```diff
@@ -264,7 +264,11 @@ class GeometryDetails extends React.Component {
                                 <span/>
                             </Col>
                             <Col xs={2}>
-                                <span className="details-circle-attribute-name"><I18N.Message msgId={"queryform.spatialfilter.details.radius"}/>{':'}</span>
+                                <span className="details-circle-attribute-name">
+                                    <I18N.Message
+                                        msgId="queryform.spatialfilter.details.radius"
+                                        msgParams={{projection: this.isWGS84() ? "°" : "m" }}/>{':'}
```

Comment:
isWGS84 is not properly the way to get the unit of measure of current projection. CoordinatesUtils.Units should do the work